mcs/class/referencesource/System/net/System/Net/webclient.cs

```diff
@@ -1805,20 +1805,9 @@ private string GetStringUsingEncoding(WebRequest request, byte[] data)
             OnOpenReadCompleted((OpenReadCompletedEventArgs)arg);
         }
         private void OpenReadAsyncCallback(IAsyncResult result) {
-#if MONO
```

Comment:
This section has been cleaned up to no longer depend on implementation internals.